Source/Resources/Content/MSGraph/DeviceManagement/DeviceConfigurations/UX - Windows Spotlight Disabled.json

```diff
@@ -8,7 +8,7 @@
       }
     }
   ],
-  "description": "Disables Windows Spotlight, a consumer oriented feature that provides tips and suggestions on the lock screen (including third party app recommendations).",
+  "description": "What is it: Disables Windows Spotlight, a consumer oriented feature that provides tips and suggestions on the lock screen, including third party app recommendations. \n Learn More: https://docs.microsoft.com/en-us/windows/configuration/windows-spotlight",
```

Comment:
**What does this do?**
Disables Windows Spotlight, which displays tips and third party marketing materials on users' lock screen. 

**Why should you use this?**
If you want to eliminate marketing materials being displayed to users. 

**What is the end user impact?**
Users will not receive third party marketing materials. 

**Learn more**
[Configure Windows Spotlight on the lock screen](https://docs.microsoft.com/en-us/windows/configuration/windows-spotlight)